src/screens/CreatePost/elements/PollItem.js

```diff
@@ -73,7 +63,7 @@ const S = StyleSheet.create({
     borderRadius: 10,
     marginVertical: 4,
     paddingHorizontal: 8,
-    borderColor: colors.holytosca,
+    borderColor: COLORS.holyTosca
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `colors.holytosca` to `COLORS.holyTosca` seems to be a simple renaming for consistency. However, ensure that the new color value is correctly defined in the `COLORS` object.
```diff
-    borderColor: colors.holytosca,
+    borderColor: COLORS.holyTosca,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->